utils/oci-runtime-tools/Makefile

```diff
@@ -0,0 +1,72 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oci-runtime-tools
+PKG_VERSION:=0.9.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE_NAME:=runtime-tools
+PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/opencontainers/runtime-tools/archive/v$(PKG_VERSION)
+PKG_SOURCE_VERSION:=ee63cfa6f66491e78f8e63c37806d7a905eb836e
+PKG_HASH:=4c2978e9097a3c93648c8ecf4f785f728044cddf717686e0b05a9244a1ad273c
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/opencontainers/runtime-tools/
+GO_PKG_LDFLAGS_X:=main.gitCommit=$(PKG_SOURCE_VERSION) main.version=$(PKG_SOURCE_VERSION)
+
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/oci-runtime-tool
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=oci-runtime-tool
+  URL:=https://github.com/opencontainers/runtime-tools
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/oci-runtime-tool/description
+  Podman: A tool for managing OCI containers and pods
+endef
+
+define Package/oci-runtimetest
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=OCI runtimetest tool
+  URL:=https://github.com/opencontainers/runtime-tools
+  DEPENDS:=$(GO_ARCH_DEPENDS) oci-runtime-tool
+endef
+
+define Build/Compile
+	$(call GoPackage/Build/Compile)
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+endef
+
+define Package/oci-runtime-tool/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/oci-runtime-tool $(1)/usr/bin
+endef
+
+define Package/oci-runtimetest/install
+	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/libexec/oci-runtime-validation-executables
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/runtimetest $(1)/usr/bin
+	( cd $(PKG_BUILD_DIR) ; $(FIND) ./validation/ -name *.go | grep -v util | \
+	xargs -I'%' -n 1 basename % .go | while read testbin; do \
+		$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$$$${testbin} \
+			$(1)/usr/libexec/oci-runtime-validation-executables/$$$${testbin}.t ; \
+	done )
+endef
+
+
+$(eval $(call GoBinPackage,oci-runtime-tools))
+$(eval $(call BuildPackage,oci-runtime-tool))
```

Comment:
Wouldn't it be better if you use plural for both?

Reply:
While the source package is called `runtime-tools` (plural) the packaged executable is called `oci-runtime-tool` (singular).
The other binaries, `runtimetest` as well as the validation executables are packaged in `oci-runtimetest`.